src/app/component/shared/formly/timezone-input.component.ts

```diff
@@ -1,67 +1,133 @@
-import { Component, OnInit } from "@angular/core";
+import { Component, OnInit, ViewChild } from "@angular/core";
+import { NgbTypeahead } from "@ng-bootstrap/ng-bootstrap";
 import { FieldType } from "@ngx-formly/core";
+import { merge, Observable, Subject } from "rxjs";
+import {
+  debounceTime,
+  distinctUntilChanged,
+  filter,
+  map,
+} from "rxjs/operators";
 
 /**
  * Timezone Input
  */
 @Component({
   // tslint:disable-next-line: component-selector
   selector: "formly-timezone-input",
-  template: ` <p>Timezone Input Placeholder</p> `,
-})
-// template: `
-//   <div class="form-group">
-//     <label *ngIf="field.templateOptions.label" [for]="field.id">
-//       {{
-//         field.templateOptions.label +
-//           (field.templateOptions.required ? " *" : "")
-//       }}
-//     </label>
+  template: `
+    <div class="form-group">
+      <label *ngIf="field.templateOptions.label" [for]="field.id">
+        {{
+          field.templateOptions.label +
+            (field.templateOptions.required ? " *" : "")
+        }}
+      </label>
+
+      <ng-template #rt let-r="result" let-t="term">
+        <ngb-highlight
+          [result]="r.currentTimeFormat"
+          [term]="t"
+        ></ngb-highlight>
```

Comment:
the screenshot in the PR shows the textual labels using the `name` value instead of `currentTimeFormat` - is that an issue with the screenshot or is this code not working as intended?

Reply:
Screenshot, i forgot to update it, my bad